### PR TITLE
chore(novu, novu-labs, framework, create-novu-app): Publish prerelease

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?
Publish pre-release versions:
* `create-novu-app@0.24.3-alpha.8`
* `novu@0.24.3-alpha.4`
* `@novu/framework@0.24.3-alpha.13`
* `novu-labs@0.24.3-alpha.1`

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: .source
Comment (rifont) [inline .source:1]: Bringing the submodule to the latest `next` commit, it was referencing a feature branch commit

File: packages/framework/package.json [hunk 1]
```diff
@@ -47,8 +47,8 @@
     "h3",
     "express"
   ],
-  "author": "Novu",
-  "license": "MIT",
+  "author": "Novu Team <engineering@novu.co>",
+  "license": "ISC",
```
Comment (rifont) [inline packages/framework/package.json:51]: Keeping consistent with other licenses in repo.

File: packages/cli-next/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@novu/novu-labs",
-  "version": "0.24.2",
-  "description": "On-Boarding Cli",
+  "name": "novu-labs",
+  "version": "0.24.3-alpha.1",
+  "description": "Novu Labs CLI. Used to sign-up, sync with Novu Cloud, and run Novu Studio.",
```
Comment (denis-kralj-novu) [inline packages/cli-next/package.json]: ✏ **suggestion:** ‏ Can we actually include a deprecation warning here as well?

**Review by denis-kralj-novu — APPROVED** [review]

File: packages/cli-next/package.json (hunk 2 above)
Comment (rifont) [reply packages/cli-next/package.json]: good call! Updated

Comment (rifont) [inline packages/cli-next/package.json]:
```suggestion
  "description": "Deprecated: use `novu` instead. Novu Labs CLI. Used to sign-up, sync with Novu Cloud, and run Novu Studio.",
```

File: packages/cli-next/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@novu/novu-labs",
-  "version": "0.24.2",
-  "description": "On-Boarding Cli",
+  "name": "novu-labs",
+  "version": "0.24.3-alpha.1",
+  "description": "Deprecated: use `novu` instead. Novu Labs CLI. Used to sign-up, sync with Novu Cloud, and run Novu Studio.",
```
Comment (rifont) [inline packages/cli-next/package.json]:
```suggestion
  "description": "DEPRECATED: use `novu` instead. Novu Labs CLI. Used to sign-up, sync with Novu Cloud, and run Novu Studio.",
```

**Pull request merged**